### DISCUSSION TITLE
/MAT/LAW70 (FOAM_TAB)

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#2535] [Q&A]
Hi everyone,

Do you guys know what is the similar material model (/MAT/LAW70 (FOAM_TAB) ) in ABAQUS? LOW DENSITY FOAM? HYPERFOAM?

Thanks

**Comment by MarianBulla** [comment]
Hi @li2qn,
yes, the *LOW DENSITY FOAM seems to be best for modelling LAW70 or LAW90 foams.
If you want to go with *HYPERFOAM, the equivalent is LAW62 in OpenRadioss.

Hope it helps you ?

Thanks and best regards,
Marian
